README.md

```diff
@@ -1,76 +1,54 @@
+follow [these instructions](#2-configuration) for install and configuration.
 
-## Getting Started
-Use GitHub's template feature to make a copy of this repository by clicking the "Use this template" button then clone outside of `$GOPATH`.
+## Table of Contents
 
-Alternatively shallow clone the repository to a directory outside of `$GOPATH` matching your plugin name:
-```
-git clone --depth 1 https://github.com/mattermost/mattermost-plugin-starter-template com.example.my-plugin
-```
+- [1. Features](#1-features)
+- [2. Configuration](#2-configuration)
 
-Note that this project uses [Go modules](https://github.com/golang/go/wiki/Modules). Be sure to locate the project outside of `$GOPATH`, or allow the use of Go modules within your `$GOPATH` with an `export GO111MODULE=on`.
+## 2. Configuration
 
-Edit `plugin.json` with your `id`, `name`, and `description`:
-```
-{
-    "id": "com.example.my-plugin",
-    "name": "My Plugin",
-    "description": "A plugin to enhance Mattermost."
-}
-```
+### Step 1 Create Mattermost App Azure (Private or Enterprise MS account)
 
-Build your plugin:
-```
-make
-```
+- Sign into [portal.azure.com](www.portal.azure.com)
+  - from the hamburger menu -> `Azure Active Directory`
 
-This will produce a single plugin file (with support for multiple architectures) for upload to your Mattermost server:
+#### Azure Active Directory
 
-```
-dist/com.example.my-plugin.tar.gz
-```
+- `App registrations`
+  - New registration - `Mattermost MS Calendar Plugin`
+- `Certificates & secrets`
+  - New client secret
+- `API permissions` -> `MsGraph` -> `calendars`
+  - add needed permissions
+  - (Read, Read.Shared, ReadWrite, ReadWrite.Shared)
+- `Authentication`
+  - Redirect URI -> `<MM_SITEURL>/plugins/com.mattermost.msoffice/oauth2/complete`
+    - For development (use ngrok.io URL)
 
-There is a build target to automate deploying and enabling the plugin to your server, but it requires configuration and [http](https://httpie.org/) to be installed:
-```
-export MM_SERVICESETTINGS_SITEURL=http://localhost:8065
-export MM_ADMIN_USERNAME=admin
-export MM_ADMIN_PASSWORD=password
-make deploy
-```
+### Step 2 Configure Plugin Settings
 
-Alternatively, if you are running your `mattermost-server` out of a sibling directory by the same name, use the `deploy` target alone to  unpack the files into the right directory. You will need to restart your server and manually enable your plugin.
+**`System Console` > `PLUGINS` > `MS Office Calendar`**
 
-In production, deploy and upload your plugin via the [System Console](https://about.mattermost.com/default-plugin-uploads).
+- [ ] (TODO: rename in plugin settings - currently `TODO:name`)
 
-## Q&A
+Personal
 
-### How do I make a server-only or web app-only plugin?
+- `Admin User IDs` - Add your sysadmin user ID
+- `tenantID` - Leave as “common"
+- `clientID` - copy from Azure App
+- `Client Secret` - copy from Azure App
```

Comment:
I think we should be explicit about where to find these. Maybe screenshots?

Reply:
@mickmister Yes, screens would be verrry helpful I suspect, based on my experience with Azure portal. ;)